astropy/coordinates/representation.py

```diff
@@ -348,11 +376,15 @@ def __repr__(self):
         prefixstr = '    '
         arrstr = _array2string(self._values, prefix=prefixstr)
 
+        diffstr = ''
+        if getattr(self, 'differentials', None):
+            diffstr = '\n (has differentials: {0})'.format(
+                ', '.join([repr(key) for key in self.differentials.keys()]))
```

Comment:
Yea, this is why I ended up agreeing with you about just showing 'has differentials'. What is shown now are the *keys* to the differentials dictionary. I'm not sure how to get rid of the ambiguity you mention - I worry that if we do `d/ds` people might get confused about what the actual key is. I think we either do something like `has differentials with keys=['s', 's2']` or just go back to `has differentials`